compiler/codegen/cg-expr.cpp

```diff
@@ -6175,7 +6175,14 @@ void CallExpr::codegenInvokeTaskFun(const char* name) {
   taskBundle   = codegenValue(get(1));
   bundleSize   = codegenValue(get(2));
 
-  args[0]      = new_IntSymbol(-2 /* c_sublocid_any */, INT_SIZE_32);
+  // We would like to remove this conditional and always do the true branch,
+  // but wanted to limit the impact of this near the release date.
+  if (localeUsesGPU()) {
```

Comment:
I'm a little surprised this makes any difference, because it looks like the tasking layers always set the requested sublocale to `c_subloc_any` anyway. Are you seeing something different?